### DISCUSSION TITLE
need build-logic explanation

### DISCUSSION THREAD

**Discussion by lfgtavora** [android/nowinandroid#1349] [Q&A]
Hi! Studying this project i miss some informations about how build-logic module works, its hard to understand whats the motivations, why was divided in many convention. There is a lack of documentation of this topic in my opinion, and its hard to follow since carry a lot of gradle stuff. A better detailed explanation should be great!

**Comment by Jaehwa-Noh** [comment]
I leave my opinion in here.
I'm not sure I am right.
The reason of divided in many convention would try to use for ```Incremental build``` and ```Build Cache``` smartly.

**Comment by SimonMarquis** [comment]
Hi @lfgtavora , the [build-logic/README.md](https://github.com/android/nowinandroid/blob/main/build-logic/README.md) file should answer most of your questions.
